mingw-w64-git/git-bash.rc

```diff
@@ -4,4 +4,4 @@ BEGIN
 	1 "SHOW_CONSOLE=1 APPEND_QUOTE=1 @@COMSPEC@@ /S /C """"@@EXEPATH@@\\usr\\bin\\bash.exe"" --login -i"
 END
 
-"MAINICON" ICON "git-for-windows.ico"
+MAINICON ICON "git-for-windows.ico"
```

Comment:
I tested this both with `MINGW_ARCH=mingw64` (GCC) and `MINGW_ARCH=clangarm64` (Clang). In both cases, the icon is set correctly:

![image](https://user-images.githubusercontent.com/17739158/193000025-f4feb3c6-061c-4fee-bf98-18ca3a75909c.png)